src/benchmarks/protocol/protocol_benchmark.cpp

```diff
@@ -0,0 +1,126 @@
+#include <benchmark/benchmark.h>
+
+#include "MumbleProtocol.h"
+#include "PacketDataStream.h"
+
+#include <limits>
+#include <random>
+#include <vector>
+
+std::random_device rd;
+std::mt19937 rng(rd());
+std::uniform_int_distribution< unsigned int > random_integer(1, 1024);
+std::uniform_int_distribution< unsigned int > random_byte(0, std::numeric_limits< Mumble::Protocol::byte >::max());
+
+constexpr int PAYLOAD_SIZE_RANGE = 0;
+
+constexpr int fromPayloadSize       = 0;
+constexpr int toPayloadSize         = 900;
+constexpr int payloadSizeMultiplier = 2;
```

Comment:
I see three different styles for `constexpr` variables across the pull request:

`VAR_NAME`
`VarName`
`varName`

Should we perhaps stick to a single one?

Reply:
Good point. I'll change it to VAR_NAME :+1: